packages/antd/test/__snapshots__/Array.test.js.snap

```diff
@@ -415,21 +434,16 @@ exports[`array fields fixed array 1`] = `
                             className="ant-input-number-input-wrap"
                           >
                             <input
-                              aria-valuemin={-9007199254740991}
+                              aria-valuenow={null}
```

Comment:
do you know why this change happened? is this because of the antd upgrade?

Reply:
I'm not quite sure since I reverted the upgrade and re-installed all the packages.🤔